web/init/src/components/shared/NavBar.jsx

```diff
@@ -144,7 +142,7 @@ export class NavBar extends React.Component {
         : "",
     );
 
-    const itemsArr = [this.getNavItems.bind(this)];
+    const itemsArr = [this.getNavItems];
```

Comment:
This still requires a bind. You are referencing the function and calling it later, which will cause the context to be lost.